doc/htmldoc/templates/index.html

```diff
@@ -0,0 +1,248 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+
+<head>
+  <title>NEST documentation index </title>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
+  <meta name="description" content="This is the documentation index for the NEST, a simulator for spiking neuronal networks.">
+  <meta name="keywords" content="">
+  <link rel="stylesheet" href="_static/css/bootstrap/bootstrap.min.css">
+  <link rel="stylesheet" href="_static/css/normalize.css">
+  <link rel="stylesheet" href="_static/css/default.min.css">
+  <script src="_static/js/highlight.min.js"></script>
+  <script>hljs.highlightAll();</script>
+  <link rel="stylesheet" href="_static/js/flexslider/flexslider.css">
+  <link rel="stylesheet" href="_static/css/custom.css">
+  <script src="_static/js/modernizr-2.6.2.min.js"></script>
+    </head>
+<body id="home">
+
+
+  <a href="#" id="gototop"><img src="_static/img/arrow-up-circle.svg"/></a>
+  <header id="header" class="wrapper clearfix">
+    <div id="banner">
+      <div id="logo">
+	<img src="_static/img/nest_docs.png" alt="logo" width="100%">
+      </div>
+  </div>
+
+    <nav id="topnav" role="navigation">
+      <div class="menu-toggle">Menu</div>
+    <ul class="srt-menu" id="menu-main-navigation">
+ <li><a href="#intro">Get started</a></li>
+	<li><a href="#topics">Learn NEST</a></li>
+	<li><a href="#backmatter">Ecosystem</a></li>
+      </ul>
+    </nav>
+  </header>
+  <script src="_static/js/tsparticles.min.js"></script>
+  <script src="_static/js/particles-app.js"></script>
+  <section id="hero" class="clearfix">
+    <div class="wrapper">
+      <div class="row">
+        <div class="grid_4">
+          <h1>Welcome!</h1>
+          <h2>NEST simulator documentation</h2>
+  <h3>
+  Click the button below to get NEST!
+  </h3>
+        <a href="installation/index.html" class="buttonlink">Install NEST</a>
+
+        </div>
+        <div class="grid_2">
+          <h3> Explore the docs </h3>
```

Comment:
I think the H1 and H2 and H3 headlines here can go away completely. They look like some filler text without meaning and most people probably know what buttons are good for ;-)

Reply:
I removed most, but I like the welcome to the documentation text, so I left that in